src/sentry/migrations/0726_apitoken_backfill_hashes.py

```diff
@@ -0,0 +1,83 @@
+# Generated by Django 5.0.6 on 2024-05-29 21:28
+
+import hashlib
+import logging
+
+from django.db import migrations, router
+from django.db.backends.base.schema import BaseDatabaseSchemaEditor
+from django.db.migrations.state import StateApps
+
+from sentry.new_migrations.migrations import CheckedMigration
+from sentry.utils.query import RangeQuerySetWrapperWithProgressBar
+
+logger = logging.getLogger(__name__)
+
+
+def backfill_hash_values(apps: StateApps, schema_editor: BaseDatabaseSchemaEditor) -> None:
+    ApiToken = apps.get_model("sentry", "ApiToken")
+    ControlOutbox = apps.get_model("sentry", "ControlOutbox")
+    try:
+        from sentry.models.outbox import OutboxCategory, OutboxScope
+        from sentry.silo.safety import unguarded_write
+        from sentry.types.region import find_regions_for_user
```

Comment:
This function uses database models directly and so will fail to run if those models are modified by a future migration.

It's a simple set of functions, so I'd recommend copying them into this migration file and passing the models from `apps.get_model` to them so that this won't break.

Reply:
Done!